composer.json

```diff
@@ -1,25 +1,66 @@
 {
-    "name": "robloach/libretro-netplay-registry",
-    "description": "libretro's NetPlay Registry",
+    "name": "ashura/libretro-netplay-registry",
+    "license": "proprietary",
     "type": "project",
-    "license": "MIT",
-    "authors": [
-        {
-            "name": "Rob Loach"
+    "autoload": {
+        "psr-4": {
+            "": "src/"
+        },
+        "classmap": [
+            "app/AppKernel.php",
+            "app/AppCache.php"
+        ]
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests/"
         }
-    ],
+    },
+    "require": {
+        "php": "~7.0",
+        "phpunit/phpunit": "3.7.*",
+        "symfony/symfony": "3.2.*",
+        "doctrine/orm": "^2.5",
+        "doctrine/doctrine-bundle": "^1.6",
+        "doctrine/doctrine-cache-bundle": "^1.2",
+        "symfony/swiftmailer-bundle": "^2.3.10",
+        "symfony/monolog-bundle": "^3.0.2",
+        "symfony/polyfill-apcu": "^1.0",
+        "sensio/distribution-bundle": "^5.0",
+        "sensio/framework-extra-bundle": "^3.0.2",
+        "incenteev/composer-parameter-handler": "^2.0",
+        "twig/twig": "^1.0||^2.0"
+    },
+    "require-dev": {
+        "sensio/generator-bundle": "^3.0",
+        "symfony/phpunit-bridge": "^3.0"
+    },
     "scripts": {
-        "test": [
-            "phpunit test --bootstrap=test/autoload.php --no-coverage",
-            "phpcs --standard=PSR2 test src raw index.php"
+        "symfony-scripts": [
+            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
+            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
+            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
+            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
+            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
+            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
+        ],
+        "post-install-cmd": [
+            "@symfony-scripts"
         ],
-        "phpcbf": "phpcbf index.php test src raw",
-        "clean": "rm -rf .*.sqlite",
-        "start": "php -S localhost:8080 # Visit http://localhost:8080"
```

Comment:
Could we get `composer start` and `composer test` back? Find them handy for local development.